server/api.js

```diff
@@ -44,23 +44,30 @@ router.get("/laptop_request", async (req, res) => {
 });
 
 router.post("/laptop_donation", (req, res) => {
-	// create a database table for 
+	// create a database table for
 	let name = req.body.name;
 	let address = req.body.address;
-	let laptops = req.body.numberOfLaptops;
-	let phonenumber = req.body.phoneNumber;
+	let number_of_laptops = req.body.numberOfLaptops;
```

Comment:
sorry to continue to be difficult. Javascript variables should have a consistent naming convention throughout. so numberOfLaptops instead of number_of_laptops. (The reason for such conventions is not because there is an absolute "right" or "wrong", but so that there is an obvious answer to "should I use camelcase or snakecase"). And then sql is different (because sql is annoying :))

Reply:
@gregdyke I have pushed changes to fix js variables back to camelCase. I'm not sure how to format using eslint. But when I save it does format it in some way.